packages/core/src/filters/FilterSystem.ts

```diff
@@ -173,6 +176,14 @@ export class FilterSystem extends System
 
         state.sourceFrame.copyFrom(target.filterArea || target.getBounds(true));
 
+        const transform = renderer.projection.transform;
+
+        if (transform)
+        {
+            // TODO: use Bounds.addFrameMatrix instead
+            this.transformAABB(transform, state.sourceFrame);
```

Comment:
We must not transform the source frame if the filter area is specified if I'm not mistaken. Only the bounds should be transformed.

Reply:
Yeah, whether filterArea is global coords or screen (after transform) is certainly a question. Consider we assign it to "renderer.screen"...

Reply:
OK, I agree, changing it. Just because i put `filterArea = renderer.screen` as an example. everywhere.